### PR TITLE
Update Mobile development.md

### PR DESCRIPTION
**Pull request by Stefterv**

### CODE REVIEW

File: en/Plugins/Getting started/Mobile development.md [hunk 1]
```diff
@@ -18,6 +18,17 @@ To disable mobile emulation, enter the following and press `Enter`:
...
 ```
 
+## Inspecting the webview on the actual mobile device
+
+### Android
+
+You can inspect Obsidian running on an Android device if you enable USB Debugging in Developer settings of Android. Then go to a chromium based browser on your desktop/laptop and navigate to chrome://inspect/. If you did everything right, if you have your phone/tablet connected to your PC via USB and the browser open at that link you should see your device pop up and it will let you run the usual devtools from there on it.
+
+### iOS
+
```
Comment (joethei) [inline en/Plugins/Getting started/Mobile development.md:28]: This won't work for versions before iOS 16.4, and needs some setup on both ends first.
You can link to: https://webkit.org/web-inspector/enabling-web-inspector/ for more instructions

**Review by joethei — CHANGES_REQUESTED** [review]

File: en/Plugins/Getting started/Mobile development.md [hunk 2]
```diff
@@ -18,6 +18,17 @@ To disable mobile emulation, enter the following and press `Enter`:
 this.app.emulateMobile(false);
 ```
 
+## Inspecting the webview on the actual mobile device
```
Comment (lishid) [inline en/Plugins/Getting started/Mobile development.md:21]: This is the wrong place to put it - the next section is still talking about mobile emulation on the desktop app.

I suggest moving this section to the very end of the document.

Comment (lishid) [reply en/Plugins/Getting started/Mobile development.md:21]: Or at least just before "Platform-specific features"

File: en/Plugins/Getting started/Mobile development.md [hunk 3]
```diff
@@ -18,13 +18,24 @@ To disable mobile emulation, enter the following and press `Enter`:
 this.app.emulateMobile(false);
 ```
 
+
```
Comment (lishid) [inline en/Plugins/Getting started/Mobile development.md:21]: Oops!

**Review by lishid — APPROVED** [review]

File: en/Plugins/Getting started/Mobile development.md (hunk 2 above)
Comment (Stefterv) [reply en/Plugins/Getting started/Mobile development.md:29]: True I missed the callout there 😅

**Pull request merged**